### ISSUE TITLE
Multiple SCM  plugin is not supported in the Multi branch project plugin

### ISSUE THREAD

**Issue by narin-kura** [mjdetullio/multi-branch-project-plugin#103]
in other orchestration tools ,we have a feature of having multiple repositories configured and branches of any 1 single Repository would be used for creation of  sub projects. But This plugin doesnt support another plugin called Multiple SCM.  

we have a complex setup 
Currently we use Accurev and GIT as source code repositories. still we are migrating. 

in Jenkins we have a plugin called Multiple SCM , which allows us to  have multple Repositories configured in single Jenkins job . 
but when i try to create Multi branch project plugin , i  do not find an option to add multiple SCM. .

In Short , we want to use  combination of Multiple SCM and Multi  branch Project plugins

**Comment by mjdetullio** [comment]
This plugin does not provide the individual SCM options, so nothing I would be implementing here.  On top of that, what you're trying to do seems like it's outside the scope of this plugin.  Either way I feel there's likely an incompatibility with the APIs of this plugin and Multiple SCM plugin.

The only reasonable option is to use Git with this plugin and then do your Accurev checkout as a build step at the beginning.  Or get hacking away on a bunch of pull requests across several plugins.  You'll probably finish your SCM migration before you finish that though.
